xmatters_connector.py

```diff
@@ -644,13 +648,24 @@ def _get_oncall_user(self, param):
 
         for res in response_json['data']:
             action_result.add_data(res)
+        i = 0
+        while i <= (len(response_json['data']) - 1):
+            for k, j in (response_json['data'][i]).items():
+                if k == 'shift':
+                    shift_count += 1
+            i += 1
```

Comment:
suggestion: what do you think about using `enumerate` instead? It should simplify this block:

![image](https://user-images.githubusercontent.com/82124762/148112850-bd0ee7da-63f4-4873-a618-596de564d73b.png)

Reply:
improved the code for shift count